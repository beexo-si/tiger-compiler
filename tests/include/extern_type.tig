/*TIGER_ARGS=../tests/include/afora.o*/
let
    type strange = int
    extern function afora(x:strange):int
    var x:=afora(5)
in
    x
end
