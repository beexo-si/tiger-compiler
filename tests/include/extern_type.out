bien!
/usr/bin/ld: /tmp/cckYuDud.o: in function `L2':
(.text+0x1a): undefined reference to `afora'
collect2: error: ld returned 1 exit status
Fail: Error al ejecutar gcc
